src/components/TradeWidget/TokenRow.tsx

```diff
@@ -210,7 +210,11 @@ const TokenRow: React.FC<Props> = ({
             <strong>
               <Link to="/deposit">Exchange wallet:</Link>
             </strong>{' '}
-            <span className="success">{formatAmount(exchangeBalanceAndPendingBalance, balance.decimals) || '0'}</span>
```

Comment:
`formatAmount` already deals with `null` inputs, this code is redundant.

Reply:
That check is for `balance.decimals`. Changed var to be more understandable.